aspnetapp/README.md

```diff
@@ -0,0 +1,27 @@
+﻿aspnet-framework:4.6.2 Sample
+====================
+
+This sample demonstrates how to dockerize an ASP.NET Web Form app that uses .NET Framework 4.6.2 by adding a `Dockerfile` to it and running the image in [Docker for Windows](https://docs.docker.com/docker-for-windows/) with a Windows Container. To complete this example you must have [Windows 10](https://www.microsoft.com/en-us/windows/get-windows-10) (or [Windows Server 2016](https://www.microsoft.com/en-us/cloud-platform/windows-server)), [Docker for Windows](https://docs.docker.com/docker-for-windows/), [Visual Studio](https://www.visualstudio.com/vs/), and [Git](https://git-scm.com/) installed.
+
+## Instructions
+
+> **Note:** You must use Windows Containers on [Docker for Windows](https://docs.docker.com/docker-for-windows/) to run this image. Be sure to check that you are properly switched to Windows Containers. Do this by opening the system tray up arrow and right clicking on the Docker whale icon for a popup menu. In the popup menu make sure you select 'Switch to Windows Containers'. 
+
+### Preparing your Environment
+
+1. Git clone this repository or otherwise copy this sample to your machine: `git clone https://github.com/dotnet/dotnet-framework-docker-samples/aspnetapp-4.6.2`
+2. Build the project locally from [Visual Studio](https://www.visualstudio.com/vs/).
+3. Navigate to this sample on your machine at the command prompt or terminal. Make sure terminal is open to the same folder that contains the Dockerfile.
+
+### Build and run the Docker image:
+
+> **Note:** The base image for this sample is [microsoft/iis](https://hub.docker.com/r/microsoft/iis/) which uses [microsoft/windowsservercore](https://hub.docker.com/r/microsoft/windowsservercore/) which is a minimal install of Windows Server 2016 and about 4 GB in size. Make sure you have enough space availiable on your drive.
+
+1. Build the Docker image: `docker build -t aspnetapp .`
+2. Run the application in the container: `docker run -d -p 80:80 aspnetapp`
+
+### View your web page running from your container
```

Comment:
I like the command-line steps, but it is also possible to simply 'Add Docker Support' to the project from Visual Studio 2017.  This enables the simple compile and run experience that ASP.NET 4.x developers are familiar with, and allows them to continue to avoid the command-line (which many of this audience prefer )